### ISSUE TITLE
[Dashboard] SQL Editor: Cannot use due to error: "Your queries will not be saved as you do not have sufficient permissions"

### ISSUE THREAD

**Issue by madebyfabian** [supabase/supabase#14425]
# Bug report

## Describe the bug

When I try to add a new (actually the first) sql editor by clicking "+ New query" nothing happens and I just get this error toast:
![Bildschirm­foto 2023-05-17 um 14 28 12](https://github.com/supabase/supabase/assets/32813692/35a58c01-ee24-4ba1-805b-e4482ff4823e) 
saying "Your queries will not be saved as you do not have sufficient permissions". 

I tried to log out and in again. Does not help. It's a freshly created project, created it a few hours ago.

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Sign into app.supabase.com
2. Click on 'SQL Editor'
3. Click on '+ New query'

## Expected behavior

I should be able to create new sql editor entries.

## Screenshots

See above

## System information

- OS: MacOS
- Browser: Chrome 112

## Additional context

–

**Comment by madebyfabian** [comment]
![Bildschirm­foto 2023-05-17 um 14 33 46](https://github.com/supabase/supabase/assets/32813692/30631acd-8a68-4fe3-a665-6327c19e3e7a)

I do get this network request that errors with 409 error at the same time when trying to add a new sql editor.

**Comment by madebyfabian** [comment]
@saltcod Unfortunately, it's still happening. I also tried to restart the whole project, but didn't helped.
I also notice various other auth-related bugs. Like for example when using the dashboard, often times I do not get the option to delete/edit rows. I have to reload the tab then to be able to delete a row. So it seems at some point Supabase always sees me as a kind of view-only user.

I just tried it in another project on my account (one that is older and also has some sql queries already created) and creating a new one also errors! So it's not just for one project. Seems something is wrong with the account.

Tried it with a different account in another project and everything works fine.
